### ISSUE TITLE
Feedback about Softmax

### ISSUE THREAD

**Issue by owhelloworld** [pytorch/pytorch#161252]
There is the following issue on this page: https://docs.pytorch.org/docs/stable/generated/torch.nn.Softmax.html
Example about softmax is too simple, without clear output, without different dimension.

**Comment by jbschlosser** [comment]
@owhelloworld if you'd like to open a PR with improvements to the docs, we may be willing to take a look.

**Comment by vishalgoyal316** [comment]
Is anyone working on it?
Can I look at this improvement item?
